java/core/src/main/scala/io/pdal/Pipeline.scala

```diff
@@ -0,0 +1,23 @@
+package io.pdal
+
+import ch.jodersky.jni.nativeLoader
+
+class Pipeline(val json: String) extends Native {
+  Pipeline // instead of java static
```

Comment:
"instead of java static" is a little confusion, perhaps

```scala
// reference the object so that the nativeLoader will load the JNI native libraries
```